tests/nordic/nrf52840-dk/common/application_code/main.c

```diff
@@ -23,72 +23,89 @@
  * http://www.FreeRTOS.org
  */
 
-
 /* FreeRTOS includes. */
 #include "FreeRTOS.h"
 #include "task.h"
 
-/* Test includes */
-#include "aws_test_runner.h"
-
 /* Demo includes */
-#include "aws_demo_runner.h"
+#include "aws_test_runner.h"
 
 /* AWS library includes. */
 #include "aws_system_init.h"
 #include "aws_logging_task.h"
 #include "aws_wifi.h"
 #include "aws_clientcredential.h"
-#include "aws_demo_config.h"
-
-/* AWS BLE headers */
-#include "aws_ble.h"
-#include "bt_hal_manager.h"
-#include "bt_hal_manager_adapter_ble.h"
 
 /* Nordic BSP includes */
+#include "bsp.h"
 #include "nordic_common.h"
-#include "nrf.h"
-#include "app_error.h"
-#include "ble.h"
-#include "ble_hci.h"
-#include "ble_srv_common.h"
-#include "ble_advdata.h"
-#include "ble_advertising.h"
-#include "ble_bas.h"
-#include "ble_hrs.h"
-#include "ble_dis.h"
-#include "nrf_ble_qwr.h"
-#include "sensorsim.h"
+#include "nrf_drv_clock.h"
+#include "nrf_log_ctrl.h"
+#include "nrf_log_default_backends.h"
 #include "nrf_sdh.h"
 #include "nrf_sdh_soc.h"
 #include "nrf_sdh_ble.h"
 #include "nrf_sdh_freertos.h"
-#include "app_timer.h"
-#include "peer_manager.h"
-#include "peer_manager_handler.h"
-#include "bsp.h"
-#include "bsp_btn_ble.h"
-#include "FreeRTOS.h"
-#include "task.h"
+#include "sensorsim.h"
+#include "ble_srv_common.h"
+#include "ble_advdata.h"
+#include "ble_advertising.h"
 #include "timers.h"
-#include "semphr.h"
-#include "fds.h"
+#include "app_timer.h"
 #include "ble_conn_state.h"
 #include "nrf_drv_clock.h"
 #include "nrf_ble_gatt.h"
-#include "nrf_ble_lesc.h"
+#include "nrf_ble_qwr.h"
+#include "ble_conn_params.h"
+#include "peer_manager.h"
+#include "peer_manager_handler.h"
+#include "fds.h"
+#include "bsp_btn_ble.h"
 
+#include "aws_ble_hal_dis.h"
 
 #include "nrf_log.h"
 #include "nrf_log_ctrl.h"
 #include "nrf_log_default_backends.h"
+#include "ble_types.h"
+#include "nrf_ble_lesc.h"
+
+#include "bt_hal_manager_adapter_ble.h"
+#include "bt_hal_manager.h"
+#include "bt_hal_gatt_server.h"
+#include "aws_ble_services_init.h"
+#include "app_uart.h"
+
+/* MQTT v4 include. */
+#include "aws_iot_mqtt.h"
+
+#include <aws_ble.h>
+#include "aws_iot_network_manager.h"
+#include "aws_ble_numericComparison.h"
+#include "SEGGER_RTT.h"
+#include "aws_application_version.h"
+#if defined( UART_PRESENT )
+    #include "nrf_uart.h"
+#endif
+#if defined( UARTE_PRESENT )
+    #include "nrf_uarte.h"
+#endif
 
 /* clang-format off */
+#define mainTEST_RUNNER_TASK_STACK_SIZE     ( configMINIMAL_STACK_SIZE * 12 )
 
 /* Logging Task Defines. */
 #define mainLOGGING_MESSAGE_QUEUE_LENGTH    ( 15 )
-#define mainLOGGING_TASK_STACK_SIZE         ( configMINIMAL_STACK_SIZE * 8 )
+#define mainLOGGING_TASK_STACK_SIZE         ( configMINIMAL_STACK_SIZE * 4 )
+
+/* BLE Lib defines. */
+#define mainBLE_DEVICE_NAME                 "nRF52840-dk"                               /**< Name of device. Will be included in the advertising data. */
```

Comment:
This could be replaced by the BLE configs defined in aws_ble_config.h

Reply:
good catch